tomlkit/items.py

```diff
@@ -664,7 +663,7 @@ def __rsub__(self, other):
 
     def _new(self, result):
         raw = str(result)
-
+        
```

Comment:
Redundant indentation.

Reply:
What line do i need to remove @frostming , i don't see the problem

Reply:
Line 666, whitespaces should be trimmed

Reply:
good now ?

Reply:
@frostming